index.js

```diff
@@ -0,0 +1,157 @@
+const show = document.querySelector(".show"),
+      startButton = document.querySelector(".start-btn"),
+      noShow = document.querySelector(".no-show"),
+      retryButton = document.querySelector(".retry-btn"),
+      writeNum = document.querySelector(".write-num"),
+      resultList = document.querySelector(".result-list"),
+      message = document.querySelector(".message");
```

Comment:
`const`를 쓰셨네요👍 이렇게 들여쓰기 해주셔도 될 것 같아요.

```suggestion
const show = document.querySelector(".show"),
  startButton = document.querySelector(".start-btn"),
  noShow = document.querySelector(".no-show"),
  retryButton = document.querySelector(".retry-btn"),
  writeNum = document.querySelector(".write-num"),
  resultList = document.querySelector(".result-list"),
  message = document.querySelector(".message");
```